docs/customization/translation.rst

```diff
@@ -15,23 +15,23 @@ For example:
 * change "Last name" into "Surname"
 * change "Add to cart" into "Buy"
 
-and many other places where you can customize the text content of pages.
+There are many other places where you can customize the text content of pages.
 
 How to customize a translation?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 In order to customize a translation in your project:
 
-1. If you don't have create the ``AppBundle\Resources\translations\messages.en.yml`` for english translations.
+1. If you don't have it yet, create ``AppBundle\Resources\translations\messages.en.yml`` for English translations.
 
 .. note::
 
-    You can create different files for different locales (languages). For example ``messages.pl.yml`` should hold only polish translations,
-    as they will be visible when the current locale is ``PL``. Check :doc:`Locales </book/locales>` docs for more information.
+    You can create different files for different locales (languages). For example ``messages.pl.yml`` should hold only Polish translations,
+    as they will be visible when the current locale is ``PL``. Check the :doc:`Locales </book/locales>` docs for more information.
 
-3. In this file configure the desired key and give it a translation.
+3. In this file, configure the desired key and give it a translation.
 
-If you would like to change the translation of "Email" into "Username" on the login form you have to
+Example: If you would like to change the translation of "Email" into "Username" on the login form you have to
```

Comment:
I agree we don't, I went back and re-read the section and the entire section is one big example, this only being part of it. This change is not needed.